RealmSwift/LinkingObjects.swift

```diff
@@ -422,6 +429,39 @@ extension LinkingObjects : RealmCollection {
     }
 }
 
+private struct LinkingObjectsHandoverMetadata {
+    var propertyName: String
+    var property: RLMProperty?
```

Comment:
It would, great point. I must've been scared away from tagged unions after the C++ changes…